docs/cep/error-handling/error-handling-wait-and-retry.md

```diff
@@ -5,54 +5,37 @@ title: Wait and Retry Example
 
 # Error Handling: Wait and Retry Example
 
-This example shows how errors are handled at the sink level with `wait and retry` mode.
+This section demonstrates the implementation of the `OnError.action='wait'` property in a wait and retry error handling strategy at the sink level.
 
-In this mode, publishing threads wait in back-off and re-trying mode, and only send the events when the connection is re-established. During this time the threads will not consume any new messages causing the systems to introduce back pressure on the systems that publish to it.
+## Wait and Retry Strategy
 
-For more information, refer to [Source Error Handling](source-error-handling.md) and [Sink Error Handling](sink-error-handling.md).
+In a scenario where connections to external systems are intermittent, the `OnError.action='wait'` property enables a sink to temporarily hold event publication until the connection is re-established. This behavior introduces back pressure, preventing new events from being consumed until the issue is resolved.
 
-Here is a simple example to publish abnormal glucose reading events to an unavailable HTTP endpoint. The error is handled by `wait and retry` mode.
+Here is an example demonstrating how to configure a sink to handle connection errors by using a wait and retry strategy:
 
 ```sql
--- Defines `GlucoseReadingStream` stream which contains events related to glucose readings.
-CREATE STREAM GlucoseReadingStream (locationRoom string,
-    locationBed string, timeStamp string, sensorID long,
-    patientFirstName string, patientLastName string,
-    sensorValue double);
+-- Defines the stream for handling glucose readings
+CREATE STREAM GlucoseReadingStream (locationRoom string, locationBed string, timeStamp string, sensorID long, patientFirstName string, patientLastName string, sensorValue double);
 
--- If `HTTP` endpoint is unavailable then threads who bring events via `AbnormalGlucoseReadingStream` wait in `back-off and re-trying` mode.
-
-@OnError(action='wait')
-CREATE SINK AbnormalGlucoseReadingStream WITH (type = 'http', OnError.action="wait", publisher.url = "http://localhost:8080/logger", method = "POST", map.type = 'json') (timeStampInLong long, locationRoom string, locationBed string, sensorID long, patientFullName string, sensorReadingValue double);
+-- Sink configuration with the OnError.action='wait' property for wait and retry strategy
+CREATE SINK AbnormalGlucoseReadingStream WITH (type = 'http', OnError.action='wait', publisher.url = "http://localhost:8080/logger", method = "POST", map.type = 'json') (timeStampInLong long, locationRoom string, locationBed string, sensorID long, patientFullName string, sensorReadingValue double);
```

Comment:
@diana-macrometa - The customer cannot use this endpoint the service's for two reasons:

1. He has not an access to the `c8cep` localhost.
2. If he wants to run such service somewhere else he needs the previous explanations you have already moved out below.

You could restore the lines for the logger's service and replaces `http://localhost:8080/logger` by `http|https://<loggers' host>:8080|443/logger`. All this depends on whether we should mention about this `siddhi` service at all.

Reply:
Whenever possible, we should avoid mentioning siddhi.

I do not understand what changes you are asking for.

Reply:
@diana-macrometa - If we'd like to have full-fledged example you should restore the section with the logger service no matter that it includes `siddhi`.
The customer could run the service anywhere, even on his local machine/laptop and access it from GDN's stream worker.
However the url cannot be `http://localhost:8080/logger` but e.g. `http://<customer's host>:8080/logger` or ``https://<customer's host>:443/logger`.

Reply:
Updated, and I put the prerequisite link back in.